tests/jballerina-unit-test/src/test/resources/test-src/types/decimal/decimal_usage.bal

```diff
@@ -64,3 +66,16 @@ class Student {
         self.height = height;
     }
 }
+
+// Decimal defaultable parameters
+function testDecimalDefaultable() {
+    decimal price = decimalDefaultable(540.5, discountRate = 0.1);
+    decimal expected = 496.73;
+    test:assertEquals(price, expected);
+}
+
+function decimalDefaultable(decimal fixedPrice, decimal tax = 10.28, decimal discountRate = 0.05) returns decimal {
+    decimal discount = fixedPrice * discountRate;
+    decimal price = decimal:sum(fixedPrice, tax, -discount);
```

Comment:
Since this unit test is to check only the defaultable behavior, shall we change this to return a simple addition result? 
Because, this test relies on langlib.decimal.